server/handlers/auth.js

```diff
@@ -0,0 +1,70 @@
+const db = require('../models');
+const jwt = require('jsonwebtoken');
+require('dotenv').config();
+
+exports.signin = async function (req, res, next) {
+      try {
+            const user = await db.User.findOne({
+                  email: req.body.email
+            });
+            const { id, username, category, avatarUrl } = user;
+            const isMatch = await user.comparePassword(req.body.password, next);
```

Comment:
why use 'next' parameter in const isMatch = await user.comparePassword(req.body.password, next)?